docker/Dockerfile.pypy

```diff
@@ -47,12 +48,14 @@ COPY b3lb/manage.py \
 COPY b3lb/loadbalancer ./loadbalancer
 COPY b3lb/rest ./rest
 
-RUN addgroup --gid $B3LBUID b3lb && \
-    adduser --uid $B3LBUID --gid $B3LBUID --home /usr/src/app --disabled-password --no-create-home --system b3lb && \
-    ln -s $(which pypy3) /usr/local/bin/python3 && \
-    ENV_FILE=.env.dev ./manage.py check --force-color && \
+#RUN addgroup --gid $B3LBUID b3lb && \
```

Comment:
Is this intentional? B3lb should (must) not run as root!

Reply:
Was intentional for docker run permissions. Returned to unprivileged user by integrating rendering task in celery-tasks without external docker startup.